Cargo.toml

```
@@ -74,7 +74,7 @@ time = "0.1" # TODO: Remove once `iron` is removed
 version = "0.15"
 features = ["with-chrono", "with-serde_json"]
 
-[target.'cfg(not(windows))'.dependencies]
+[target.'cfg(linux)'.dependencies]
```

Currently this is never true.
```suggestion
[target.'cfg(target_os = "linux")'.dependencies]
```